app.rb

```diff
@@ -0,0 +1,19 @@
+require 'sinatra/base'
+require_relative 'chitter'
+
+class ChitterSite < Sinatra::Base
+  get '/' do
+    erb :index
+  end
+
+  post '/chitters' do 
+    Chitter.add(params[:peep])
+    redirect('/chitters/all')
+  end
+  
+  get '/chitters/all' do
```

Comment:
Is there a way to add this to the main site, or add a link so I can see the peeps, or go back to add a peep without physically typing '/' in the URL bar?